### ISSUE TITLE
[Frostfire Bolt] - dot

### ISSUE THREAD

**Issue by UlicaTV** [gamefreedomgit/Maelstrom#3346]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
on fire mage with [Glyph of Frostfire] must apply a dot that 1x-2x-3x that gives damage to the target, but after reaching the 3x stacks accest dot does no damage if I continue to give [Frostfire Bolt], starts to deal damage only when I stop casting.

**How to reproduce:**
put [Glyph of Frostfire] on a mage wire and cast 20 [Frostfire Bolt] apply 3x dot but it won't do any damage during the cast, you have to cast the 20 [Glyph of Frostfire] consecutively without stopping.

**How it should work:**
[Frostfire Bolt]'s dot should deal damage regardless of whether I cast or not once it starts stacking on the target

**Database links:**
![fire dot](https://user-images.githubusercontent.com/132683111/236488455-4295211e-9d5a-4190-8e80-d31570bcbd53.PNG)

![fire dot](https://user-images.githubusercontent.com/132683111/236488261-5e06cd8f-65e6-4b15-8456-321e08de837a.PNG)
![dot](https://user-images.githubusercontent.com/132683111/236489148-3c0d9bfa-0c3d-4023-b471-e5ff081db08e.PNG)
![stop](https://user-images.githubusercontent.com/132683111/236491342-fd97be36-aa0d-4407-b92f-07dff62bd44d.PNG)
https://youtu.be/7Oy0YTEJAgs

**Comment by UlicaTV** [comment]
youtube link test - https://youtu.be/7Oy0YTEJAgs

**Comment by staR1336** [comment]
Not a bug. This behaviour happens with every kind of dot. When you refresh the duration of the dot by reapplying it, it simply means that you are not allowing the dot to do any kind of damage because it won't reach the tick to do damage.

You can replicate this behaviour by spamming Living Bomb and see for yourself

Closing!

**Comment by triceratopsrider** [comment]
How did this guy got a tester role? He has clearly no idea how game works. 
https://m.youtube.com/watch?v=fGw0oLq2Tew&pp=ygUSRnJvc3RmaXJlIGJvbHQgNC4y

**Comment by staR1336** [comment]
> How did this guy got a tester role? He has clearly no idea how game works. 
> 
> https://m.youtube.com/watch?v=fGw0oLq2Tew&pp=ygUSRnJvc3RmaXJlIGJvbHQgNC4y
> 
> 

Who hurt you my guy? 

Ill look into it once again,
So that you might be able to enjoy the 300damage tick.

**Comment by BalanCV** [comment]
@staR1336 
In cataclysm, dots are no longer clippable like in wrath. They will continue to tick on their own timer even if you refresh them. This is the case for most dots, the example you gave with living bomb is an exception to this rule ( in retail 4.1 you could not even refresh it before the timer the game didn't let you)
You can test out your own example by spamming warlock corruption on a target and you'll see it still ticks.

Think about it logically, what would even be the point of the dot part of the glyph if spamming frostfire causes it not to tick?

Also I'm unrelated to this thread and only felt the need to add because of your snarky satirical remarks to this player. I understand that a large number of reports are wrong, but you also have to understand bug reports are a free favor the players are doing for the server regardless of their own interest in it and deserve a tiny bit of respect for it.

Basically, if you represent the staff in any way, act like it. Peace!

**Comment by UlicaTV** [comment]
as far as I know, there's no way it can work like this, he has to give at least one tick between my helmets and if I keep reapplying it and it stays with 3x stack, he still has to give the related damage for 3 stacks
